source/help/messaging/formatting-text.rst

```diff
@@ -242,21 +249,23 @@ Inline image with hover text
     :name: Mattermost Icon
 
 Inline image with link
```

Comment:
These are technically sub sections of in-line images so the text styling should probably reflect that